package.json

```diff
@@ -7,13 +7,15 @@
     "babel-node": "babel-node --presets=env",
     "dev": "nodemon --exec npm run babel-node index.js",
     "start": "node index.js",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "cross-env process.env.NODE_ENV=test mocha --compilers js:babel-core/register ./server/tests/**/'*.test.js' --recursive --timeout 50000 --exit || true",
```

Comment:
Please remove `--timeout`, can always put this there when we need it. 
We don't have a use case at the moment.